Examples/KeyboardSupport-iOSExample/ToolbarProvidingUITextView.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ToolbarProvidingUITextView.swift
+//  KeyboardSupport-iOS
+//
+//  Created by John Davis on 1/18/19.
+//  Copyright © 2019 Bottle Rocket. All rights reserved.
+//
+
+import UIKit
+
+/// UITextView, when used in with the `KeyboardAutoNavigator` has issues accepting it's toolbar being set in the UITextViewDidBeginEditing notification handler.
+/// To workaround this behavior, UITextViews need to set their own inputAccessoryView properties
+class ToolbarProvidingUITextView: UITextView {
```

Comment:
I agree we should include a blurb in the `README` mentioning that UITextViews used with the AutoNavigator need some additional attention (and what that attention should be). 

What if we removed the overloaded `init?(decoder)`  and moved forward with setting the inputAccessoryView in `viewDidLoad`? Our README could highlight this solution used in the sample code, and also mention that it will need to occur for UITextViews that are instantiated for use in a UITableViewCell. 

We could combine it with a static variable and use that in the two spots where the toolbar needs to be set.

Reply:
File Removed